src/callbacks.jl

```diff
@@ -98,11 +99,13 @@ ContinuousCallback(condition,affect!,affect_neg!;
                    rootfind=true,
                    save_positions=(true,true),
                    interp_points=10,
+                   dtrelax=1.0,
```

Comment:
I guess it's not a good idea to enforce Float64 here, could you use
```suggestion
                   dtrelax=1,
```
instead?

Reply:
Otherwise it will automatically upscale dt to Float64, so it'll cause failures for example if someone was solving with Float32. `1` is much safer.

Reply:
understood, mde the changes